.github/workflows/ci.yml

```diff
@@ -5,6 +5,7 @@ on:
       - 'main'
       - '*-maintenance'
       - '*-dev'
+      - '*'
```

Comment:
You will have to remove this when you've finished work on it, as it causes double builds on our pipeline, your PR is not running because of Github rules surrounding first time contributors, we will have to hit the button each time you force push.